src/main/java/frc/robot/RobotContainer.java

```diff
@@ -119,39 +107,21 @@ private void configureDefaultCommands() {
     private void configureButtonBindings() {
 
         { // Xbox controller button bindings.
-            Xbox.b.whileHeld(new BackAndShootCargo2(
-                    shooter, hood, conveyor, flap,
-                    () -> 0)).whenInactive(() -> shooting = false);
+            Xbox.lt.whileActiveContinuous(new PickUpCargo(conveyor, flap, intake, 0.7, () -> Utils.map(MathUtil.clamp(Math.hypot(swerve.getChassisSpeeds().vxMetersPerSecond, swerve.getChassisSpeeds().vyMetersPerSecond), 0, 4), 0, 4, 0.25, 0.1)));
+
+            Xbox.lb.whileHeld(new Outtake(intake, conveyor, flap, shooter, hood, () -> false));
+            Xbox.rb.whileHeld(new Convey(conveyor, -Constants.Conveyor.SHOOT_POWER));
+
             Xbox.x.whenPressed(intake::toggleRetractor);
-//            Xbox.x.whileHeld(()->shooter.setVelocity(3600));
-//            Xbox.y.whenPressed(new RunCommand(() -> shooter.setVelocity(3530.0), shooter).withInterrupt(Xbox.rt::get));
             Xbox.y.whenPressed(new InstantCommand(() -> warmUpShooting = !warmUpShooting));
             Xbox.a.whileHeld(() -> playWithoutVision = true).whenReleased(() -> playWithoutVision = false);
 
             Xbox.leftPov.whileActiveOnce(new InstantCommand(hood::toggle));
             Xbox.downPov.whileActiveOnce(new LowGoalShot(shooter, flap, hood));
             Xbox.upPov.whileActiveContinuous(new JoystickPowerHelicopter(helicopter, () -> -0.1));
 
-            Xbox.rt.whileActiveContinuous(new BackAndShootCargo(
-                    shooter, hood, conveyor, flap,
-                    () -> 0)).whenInactive(() -> shooting = false);
```

Comment:
Do you think it's a good idea to have Ma'ayan shoot the cargo? If the adjustment doesn't work it could cause problems.

Reply:
Well, I also thought about it. Paulo recommended it and I also think it helps with the flow of the game. All Maayan has to do is press and hold once until balls go in. We'll see how well that, and the adjustment work. If the adjustment doesn't work we'll find safeties or split it. And if Maayan prefers it, we'll let you do it